### ISSUE TITLE
lib.haxe.org  weekly downloads ?

### ISSUE THREAD

**Issue by flashultra** [HaxeFoundation/haxe.org#343]
At the moment is available stats base on version  and include all downloads for that version ( at example  https://lib.haxe.org/p/haxe-crypto/versions/ ) 
 Is it possible to be added weekly download statistics similar at  npmjs ? 
Example: https://www.npmjs.com/package/crypto-js

**Comment by markknol** [comment]
No, this is at the moment not possible. How it works afaik is that we increase the download counter per version on each install and that's all information we have. We don't store specific date related install statistics.

**Comment by flashultra** [comment]
Also stats here https://lib.haxe.org/p/haxe-crypto/versions/  looks a little bit strange
Dates are 2016/03 to  2016/12 , but last red dot ( version ) is somewhere in 2017 ( v. 0.0.7  - 1 year ago ) 
and have 51127 downloads.
Looks like a bug.

**Comment by markknol** [comment]
Hm I see what you mean, the last dot/latest release should be at current date.

**Comment by flashultra** [comment]
@andyli  Do you think is useful to have weekly download statistics for lib.haxe.org ? ( similar to npmjs )

**Comment by andyli** [comment]
I think so. PR welcome :)

**Comment by flashultra** [comment]
Where I can find information about Version.manager in Repo.hx ( haxelib) ? 
Obviously information for downloads  came from : 
  allVersions: info.versions,  where  var info = new Repo().infos(projectName); and in Repo.hx had 
var vl = Version.manager.search($project == p.id); and downloads : v.downloads, 
but I can't see Version.manager

**Comment by andyli** [comment]
It's a [record-macros](https://github.com/HaxeFoundation/record-macros) (previously [spod](http://old.haxe.org/manual/spod)) thing.
